datamegh/api/storage/s3/s3.py

```diff
@@ -0,0 +1,56 @@
+"""Class to access AWS S3 service extends base class AWS"""
+
+from boto3.s3.transfer import S3Transfer
+from boto3.exceptions import S3UploadFailedError
+from botocore.exceptions import ClientError
+
+from datamegh.commons.aws import get_aws_client
+from datamegh.util.logging import get_logger
+
+logger = get_logger("storage:s3")
+
+
+def upload_to_s3(file_path, bucket_name, s3_object_name):
```

Comment:
Be concise and pythonic - this is already under `s3` module so just this would do.

```suggestion
def upload(file_path, bucket_name, s3_object_name):
```

And you can do this:
```py
from datamegh.api.storage import s3

s3.upload(...)
```